examples/gno.land/p/demo/json/node.gno

```diff
@@ -480,31 +497,49 @@ func ObjectNode(key string, value map[string]*Node) *Node {
 
 // IsArray returns true if the current node is array type.
 func (n *Node) IsArray() bool {
+	if n == nil {
+		return false
+	}
```

Comment:
I added a `nil` check because I confirmed that comparing with nil crashes the code.